modules/post/windows/gather/packrat_credentials.rb

```diff
@@ -0,0 +1,263 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  # this associative array defines the artifacts known to PackRat
+  APPLICATION_ARRAY = JSON.parse(
+      File.read(File.join(Msf::Config.data_directory, 'packrat', 'artifacts.json')),
+      symbolize_names: true
+    )
+
+  include Msf::Post::File
+  include Msf::Post::Windows::UserProfiles
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'         => 'Windows Gather Application Artifacts (PackRat)',
+      'Description'  => %q{
+        This module extracts artifcats from a large list of applications
+        and can extract credentials storing content in loot. Full list in
+        module documentation.

```

Comment:
~~`Full list in module documentation.`~~ -> `The full list of supported applications is in the module documentation.`